OrbitClientProtos/capture_data.proto

```diff
@@ -106,6 +122,8 @@ message CaptureInfo {
   map<uint64, FunctionStats> function_stats = 9;
   repeated TracepointInfo tracepoint_infos = 10;
   repeated TracepointEventInfo tracepoint_event_infos = 11;
+  ProcessInfo process = 13;
+  repeated ModuleInfo modules = 14;
```

Comment:
ok, looks like moving these dependencies and CaptureInfo out of proto is much larger task, lets leave it as is.